source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -377,6 +425,13 @@ export class Carousel extends Container {
         this.rtl = this.isRtl();
         this.applyRTL(carouselContainer);
 
+        if (this.isValidRenderedPageIndex(this.initialPageIndex)) {
```

Comment:
I think everything looks good, but we're seeing this issue in the designer again now:

![carousel](https://user-images.githubusercontent.com/98650930/194941939-17047e95-6068-4079-9822-2594b60b6c3c.gif)

We were using `_currentIndex` to store the state of the designer since we rerender the card every time a change is made in design mode.

With the new change, we are resetting to the initial page index each time.

Should we add a second variable to track the designer index, and use it instead of `initialPageIndex` if we're in design mode?